### PR TITLE
Bottom bar curved

### PR DESCRIPTION
**Pull request by TusharSharma651**
Customisable Curved Bottom bar. All the props have been added. Readme added.

### CODE REVIEW

File: src/components/CurvedBottomBar/Readme.md [hunk 1]
```diff
@@ -0,0 +1,177 @@
+### Example
```
Comment (mashish584) [inline src/components/CurvedBottomBar/Readme.md:1]: 1. Keep readme as short as possible
2. Declare props of component in table structure with 3 columns i.e.
      - Prop -> Name of prop
      - Type -> Type of prop
      - Description -> Description related to prop. 
      
Please checkout root readme file for example. Use any online markdown editor.

File: src/components/CurvedBottomBar/index.tsx [hunk 2]
```diff
@@ -1,240 +1,288 @@
...
+function TabsShape({
+	tabWidth,
+	index,
+	activeCurvePoints,
+	middleCurvePoints,
+	shadowStyle,
+	activeCurveRadius,
+	middleCurveRadius,
+	fillColor,
+}: any) {
```
Comment (mashish584) [inline src/components/CurvedBottomBar/index.tsx]: Use typescript strictly. Pass types here for props.

File: src/components/CurvedBottomBar/index.tsx [hunk 3]
```diff
@@ -1,240 +1,288 @@
...
+	const d = useMemo(() => {
```
Comment (mashish584) [inline src/components/CurvedBottomBar/index.tsx]: variable name should be self-explainatory.

File: src/components/CurvedBottomBar/index.tsx [hunk 4]
```diff
@@ -1,240 +1,288 @@
...
+			<View style={{ flexDirection: "row" }}>
+				{routes.map((tab, key) => {
+					if (key === Math.floor(routes.length / 2)) {
+						return (
+							<View
+								style={{
+									width: middleCurveRadius * 2,
+									height: NAVIGATION_BOTTOM_TABS_HEIGHT,
+								}}
+								key="logo"
```
Comment (mashish584) [inline src/components/CurvedBottomBar/index.tsx]: use some uniq identifier instead of logo

**Review by mashish584 — CHANGES_REQUESTED** [review]
Please review the comment & also add a working example in App.tsx like we've for button and bottom sheet component. 

We're going to add examples soon in project example directory if someone want to take reference.

File: src/components/CurvedBottomBar/Readme.md (hunk 1 above)
Comment (TusharSharma651) [reply src/components/CurvedBottomBar/Readme.md:1]: Regarding adding to App.tsx, we need to create a whole navigation structure with a bottom bar in this project. I think that would be a good idea as we will be using it as NPM package.

**Pull request closed without merging**